core/tests/protractor_desktop/adminMiscTab.js

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for admin page functionality.
+ */
+
+var general = require('../protractor_utils/general.js');
+var users = require('../protractor_utils/users.js');
+var AdminPage = require('../protractor_utils/AdminPage.js');
+var TopicsAndSkillsDashboardPage =
+  require('../protractor_utils/TopicsAndSkillsDashboardPage.js');
+var TopicEditorPage = require('../protractor_utils/TopicEditorPage.js');
+
+describe('Admin misc test tab', function() {
+  var adminPage = null;
+  var topicsAndSkillsDashboardPage = null;
+  var topicEditorPage = null;
+  var topicId = null;
+  var allowedErrors = ['Failed to load resource', 'Object', 'Entity',
+    '500', 'encode'];
+
+  beforeAll(async function() {
+    adminPage = new AdminPage.AdminPage();
+    topicsAndSkillsDashboardPage = (
+      new TopicsAndSkillsDashboardPage.TopicsAndSkillsDashboardPage());
+    topicEditorPage = new TopicEditorPage.TopicEditorPage();
+
+    await users.createAndLoginAdminUser(
+      'miscTabTester@miscTab.com', 'miscTabTester');
+
+    await topicsAndSkillsDashboardPage.get();
+    await topicsAndSkillsDashboardPage.createTopic('miscTabTopic',
+      'A topic to test the misc tab', false);
+    await browser.getCurrentUrl().then((url) => {
+      var startIndex = url.split('/', 4).join('/').length + 1;
+      topicId = url.substring(startIndex, url.length - 2);
+    });
```

Comment:
Why not just index into the result of `.split()`?

Reply:
I think `split()` returns an array of strings, so I'm not quite sure what you mean here?

Reply:
I think the ID should be one of those strings in the array returned by `split()`. For example:

https://github.com/oppia/oppia/blob/3ea28b765795c7eccbcf920dc5c79c3aa574c880/core/tests/protractor_desktop/topicAndStoryEditor.js#L61

Reply:
Ah, I see. Done!